config/initializers/session_store.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: "_velum_session"
+Rails.application.config.session_store :cookie_store,
+  key:    "_velum_session",
+  secure: Rails.env.production?
```

Comment:
OK - then for this case we should ignore the test environment.

My concern about leaving this off for dev is that this is how we do work on velum, and we may not see an error while doing other development that will only show in a production environment.

The environments are different - but we should limit the differences to only what is needed for each environment type.